sdk/swap-service/src/implementation/SwapManager.ts

```diff
@@ -0,0 +1,74 @@
+import { ChainInfo, ChainId } from '@summerfi/sdk-common/chains'
+import { Address, Percentage, TokenAmount } from '@summerfi/sdk-common/common'
+import { Maybe } from '@summerfi/sdk-common/utils'
+import { ISwapProvider, ISwapManager, SwapData, SwapProviderType } from '~swap-service/interfaces'
+
+export type SwapManagerProviderConfig = {
+  provider: ISwapProvider
```

Comment:
Don't you worry that provider could be confused with blockchain provider?

Reply:
Well, it is Swap provider. An RPC provider, or blockchain providers are other types of providers. I don't think we should stop using that word just because there are other providers around. In any case I'm happy to change it, but I couldn't come up with a better name. Jacob suggested Client, but that exposes a specific implementation and I don't think it is right.